### DISCUSSION TITLE
Hide people in the watchlist

### DISCUSSION THREAD

**Discussion by ghost** [community/community#50894] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is there a way to make the watchlist invisible on a private repository?

I have a private repository and gave read-only access to some customers, they are watching the repository in order to get notifications of any changes but they have requested to hide their usernames from the watchlist. Some of the people watching the repository are from different companies and that is the main reason they want to hide from the watchlist but keep getting notifications in case there is any change in the repository.

Is there a way to achieve that?

**Comment by enviGit** [comment]
### Hi [BR-JorgeCenteno](https://github.com/BR-JorgeCenteno)

As far as I know, there is no way to hide a user from the watchlist of a repository on GitHub. The watchlist is intended to keep track of users who are interested in a repository and receive notifications about it.

However, you could suggest to your customers to use a secondary account or an organization account to watch the repository, and use an alias or pseudonym for the account name. This way, they can still receive notifications while keeping their identity hidden from other watchers. 

Another option could be to use an external notification service, such as email, to receive notifications about changes in the repository, without having to watch it on GitHub.

**Comment by Anmol-Baranwal** [comment]
Unfortunately, there is currently no way to hide the watchlist on a private repository in GitHub. The watchlist is a standard feature on GitHub that allows users to receive notifications for changes to a repository they are interested in. When someone watches a repository, their username is added to the watchlist, which is visible to anyone with access to the repository.

However, you can suggest to your customers that they use the "ignoring conversations" feature on GitHub. This allows them to ignore notifications from specific conversations, which would include notifications from your repository. By ignoring the conversations, they would still be able to receive notifications in case there is any change in the repository, but their usernames would not appear on the watchlist.

To ignore conversations, they can go to the notification settings for your repository, click on "Ignore" next to the conversation they want to ignore, and select "Ignore this conversation." They can also choose to ignore all conversations in the repository by selecting "Ignore this repository." > Try this to see, if this can resolve the issue.
